mysql/mysql_test.go

```diff
@@ -40,6 +40,7 @@ func (t *mysqlTestSuite) TestMysqlGTIDInterval(c *check.C) {
 
 	i, err = parseInterval("1-2")
```

Comment:
I see no difference between this test and the first. Maybe we should delete this test?

Reply:
there is none, I didn't notice the earlier test. I deleted it.